### PR TITLE
Better image ci

### PR DESCRIPTION
**Pull request by mikalstill**
Draft until https://github.com/shakenfist/client-python/pull/29 merges.

### CODE REVIEW

File: deploy/shakenfist_ci/tests/test_images.py [hunk 1]
```diff
@@ -8,7 +10,26 @@ def test_cache_image(self):
...
+        # exist at the moment.
+        cache = {}
+        start_time = time.time()
+        while time.time() - start_time < 300:
+            cache = {}
+            for img in self.system_client.get_images():
+                cache.setdefault(img['url'], [])
+                cache[img['url']].append(img)
+
+            self.assertEqual(1, len(cache.get(url)))
```
Comment (mandoonandy) [inline deploy/shakenfist_ci/tests/test_images.py:28]: Would `self.assertIn(url, cache)` be better?

**Review by mandoonandy — CHANGES_REQUESTED** [review]
Small suggestion only.

File: deploy/shakenfist_ci/tests/test_images.py (hunk 1 above)
Comment (mikalstill) [reply deploy/shakenfist_ci/tests/test_images.py:28]: Done.

**Comment by mandoonandy** [comment]
Is this ready to merge?

**Review by mandoonandy — APPROVED** [review]

**Pull request merged**